src/static/css/custom.scss

```diff
@@ -593,4 +593,27 @@ section#tags {
         @extend .col-md-4;
         @extend .col-lg-3;
     }
+}
+
+section#partner {
+    @extend .py-5;
+    @extend .text-light;
+
+    h1 {
+        @extend .text-dark;
+        @extend .text-center;
+    }
+
+    img.logo {
+        @extend .mt-3;
+        @extend .mb-3;
+        @extend .mx-4;
+        @extend .text-dark;
+        @extend .d-inline-flex;
+        @extend .flex-row;
+        @extend .flex-wrap;
+        @extend .align-content-center;
```

Comment:
Attention, ces classes sont sans effets car elles sont censées être affectées au conteneur parent. C'est le conteneur qui doit recevoir le style `display: flex`, ce qui active le layout flexbox et permet d'utiliser les fonctionnalités pour aligner / espacer le contenu. En l'état actuel des choses, les logos ne sont pas centrés.

Un très bon guide sur Flexbox : https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Autre remarque : hériter les classes bootstrap permet une meilleur maintenabilité du code quand ces classes offrent des raccourcis vers des ensembles de valeurs, ou vers des variables.

Ex : `@extend .my-3` permet de spécifier une marge verticale dont la hauteur peut être changée grâce à une variable.

`@extend .d-lg-flex` permet d'appliquer le style `display: flex` uniquement pour les tailles d'écrans supérieures à `lg`.

En revanche, certaines classes sont des simples raccourcies destinées à être affectées dans le html, procédé que nous n'utilisons pas. Dans ce cas, il est plus simple de simplement configurer le style dans le css.

Ex : 

`@extend .d-flex`

n'apporte rien par rapport à : 

`display: flex`